src/mc_solver/QPSolver_setContacts.cpp

```diff
@@ -0,0 +1,25 @@
+#include <mc_solver/QPSolver.h>
+
+#include <mc_control/MCController.h>
+
+namespace mc_solver
+{
+
+void QPSolver::setContacts(const std::vector<mc_rbdyn::Contact> & contacts)
+{
+  if(controller_)
+  {
+    mc_rtc::log::warning("[MC_RTC_DEPRECATED] Prefer using addContact/removeContact for contacts manipulation");
```

Comment:
`mc_rtc::log::deprecated`? It'd be great if it stated that this should be used instead of `QPSolver::setContacts`, otherwise it makes it tricky to figure out where this deprecation warning comes from.